components/pkg-export-helm/defaults/HelmChartFile.hbs

```diff
@@ -1,5 +1,5 @@
 name: {{name}}
 version: {{version}}
-{{#if description}}
+{{~ #if description}}
```

Comment:
Oh nice... TIL about the `~` operator 👍

Reply:
Also, if this was a copy-paste from other exporters, it'd be nice to go back and make this change there, too.

Reply:
Yeah, PR coming up for k8s change very soon. Stay tuned. :)